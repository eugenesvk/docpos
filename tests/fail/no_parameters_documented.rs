use doxidize::*;

#[doxidize]
/// no function parameters documented
/// and some more
pub fn add(first: i32, second: i32) -> i32 {
    first + second
}

pub fn main() {}
