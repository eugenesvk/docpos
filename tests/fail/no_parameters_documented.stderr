error: Function has no documented parameters or generics.
       Document at least one function parameter or generic.
 --> tests/fail/no_parameters_documented.rs:6:8
  |
6 | pub fn add(first: i32, second: i32) -> i32 {
  |        ^^^
